docs/examples/coffee_machine.md

```diff
@@ -1,57 +1,31 @@
 # Coffee Machine Example
 
-This example shows how to train a model to predict the behavior of a coffee machine using an automaton. 
+This example shows how to train a model to predict the behavior of an automaton. 
 Inspired by the work of Steffen et al.[^1], we consider a simple coffee machine.
 LearnLib[^2], a framework for automata learning written in Java, is used for the inference of the automaton.
 
 !!! todo
 
-    This explanation is outdated, we no longer use `experiment.yaml` but `run.py` to run experiments.
-    Update the explanation accordingly.
+    This explanation is outdated. The automaton example has to be refined using time series data.
 
 ## Run this example
 
-There are two options to run this example.
-
 ### Docker Container
 
-For this option, the *backend* for the learner is specified to *docker*. Example snippet from *experiment.yaml*:
-```yaml
-learners:
-  - name: Automaton Learner
-    class_path: agenc.learners.grpc.GrpcLearner
-    arguments:
-      backend: docker
-      image_name: collaborating.tuhh.de:5005/w-6/forschung/agenc/agenc/java-automata-learner:latest
-```
-
 The external learner written in Java is containerized for this example. Thus, Docker must be installed to run it.
 Follow the instructions on the [official Docker page](https://docs.docker.com/get-docker/) to install it.
 The image for this example is stored in the TUHH image registry. Authentication is required for this.
 Log in to the registry using `docker login collaborating.tuhh.de` and provide your credentials when asked.
-Afterwards use the command line interface run the experiment
-
-```bash
-agenc --experiment experiment.yaml
-```
+Afterwards run the `run.py` for this example.
 
 The required images will be automatically retrieved, and a container will be started.
 
-### Local Execution
-
-For this option, the *backend* for the learner is specified to *none*. Example snippet from *experiment.yaml*:
-```
-learners:
-  - name: Automaton Learner
-    class_path: agenc.learners.grpc.GrpcLearner
-    arguments:
-      port: 8080
-      ip: "localhost"
-```
+## Local Execution
 
+It is also possible to directly run the Java server locally.
```

Comment:
Please put one line per sentence. This makes line based version diff much easier to read and maintain

Reply:
Done. Also for all (if I didn't miss something) other docu files.